### ISSUE TITLE
Delimiters should support scripts

### ISSUE THREAD

**Issue by gsomix** [ForNeVeR/wpf-math#62]
`\left.\frac{x^3}{3}\right|_0^1`

![del_scripts](https://cloud.githubusercontent.com/assets/832449/23340671/d229a76a-fc53-11e6-904c-81a77033d410.PNG)

**Comment by ForNeVeR** [comment]
Another example from #73:

```tex
d_{f,t}(u,v) = \sum_{x, y}\left(f(x, y) - t(x - u, y - v)\right)^2
```

**Comment by ForNeVeR** [comment]
Fixed by #74,
